internal/docgen/docgen.go

```diff
@@ -65,18 +63,17 @@ func Output(fileName string, content string) error {
 	outputStream := os.Stdout
 	var err error
 	if fileName != "" {
-		outputStream, err = os.OpenFile(fileName, os.O_WRONLY|os.O_CREATE|os.O_TRUNC, 0600) // #nosec
+		outputStream, err = os.OpenFile(fileName, os.O_WRONLY|os.O_CREATE|os.O_TRUNC, 0o600) // #nosec
```

Comment:
@gabriele-wolfox why is the filemode explicitly called out as `0o600` instead of regular `0600` ?